src/test/crud.test.js

```diff
@@ -0,0 +1,18 @@
+import { add, remove } from '../crud.js';
+
+jest.mock('../todoList');
+
+describe('test for todo list function', () => {
+  test('test for add funtion', () => {
+    add('hello');
+    add('goodbye');
+    add('bye');
+    const storage = JSON.parse(localStorage.getItem('todoList'));
+    expect(storage).toHaveLength(3);
+  });
+  test('test for remove funtion', () => {
+    remove(0);
+    const storage = JSON.parse(localStorage.getItem('todoList'));
+    expect(storage).toHaveLength(2);
+  });
+});
```

Comment:
Great job with testing for the local storage. However, you also need to mock HTML to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM.

Reply:
updated